admin/sql/create_tables.sql

```diff
@@ -104,4 +104,27 @@ CREATE TABLE follow_list (
 );
 ALTER TABLE follow_list ADD CONSTRAINT follow_list_name_creator_key UNIQUE (name, creator);
 
+CREATE TABLE recommendation.cf_recording (
+  id                  SERIAL, -- PK
+  user_id             INTEGER NOT NULL, --FK to "user".id
+  msid                UUID NOT NULL,
+  recommender_id      INTEGER, --FK to recommendation.recommender.id
```

Comment:
should this also be NOT NULL?

Reply:
Not sure. Whilst pushing recordings in the table, we won't have a recommender script, so I guess it will be left blank till we have a recommender script. What do you think?

Reply:
Ohh! I think we just found a bug! recommender_id should not be in this table -- it should only be in the join table, where it already is.